src/integrationTest/java/org/opensearch/test/framework/audit/AuditLogsRule.java

```diff
@@ -122,16 +135,35 @@ private void whenTimeoutOccurs(String methodName) {
     private void afterTest() {
         TestRuleAuditLogSink.unregisterListener();
         this.currentTestAuditMessages = null;
+        this.currentTransportTestAuditMessages = null;
     }
 
     private void beforeTest(String methodName) {
         log.info("Start collecting audit logs before test {}", methodName);
         this.currentTestAuditMessages = synchronizedList(new ArrayList<>());
+        this.currentTransportTestAuditMessages = synchronizedList(new ArrayList<>());
         TestRuleAuditLogSink.registerListener(this);
     }
 
     public void onAuditMessage(AuditMessage auditMessage) {
-        currentTestAuditMessages.add(auditMessage);
-        log.debug("New audit message received '{}', total number of audit messages '{}'.", auditMessage, currentTestAuditMessages.size());
+        if (auditMessage.getAsMap().keySet().contains("audit_transport_headers")) {
+            if (log.isDebugEnabled()) {
+                log.debug(
+                    "New transport audit message received '{}', total number of transport audit messages '{}'.",
```

Comment:
Will the updated tests still pass with different cluster configurations? i.e. more data nodes added to the cluster?

I like separating out the message types here which can differentiate types of audit messages.